wiki-docs/docs/interfaces/validators/validators.md

```diff
@@ -0,0 +1,45 @@
+The `IValidator` interface represents a validator and its reward pool for delegators on the childchain. This user guide will explain the structs that make up the interface.
```

Comment:
Remove this file. It is no longer needed.

Reply:
done